NEWS.md

```diff
@@ -6,7 +6,7 @@
 
 * `create_package()` and `create_project()` return a normalized path, even if target directory does not pre-exist (#227, #228).
 
-* "Tidy" GitHub templates are placed in a `.github/` subdirectory (#165 @batpigandme)
+* "Tidy" GitHub templates are placed in a `.github/` subdirectory (#165 @batpigandme).
```

Comment:
I know I wrote this sentence. But perhaps it should say "tidyverse GitHub templates". The templates themselves are neither tidy nor untidy.